sort the includes to make sure they are always listed in the same order
to make builds reproducible

Index: sunpinyin-2.0.4+git20160225.844ad0f/SConstruct
===================================================================
--- sunpinyin-2.0.4+git20160225.844ad0f.orig/SConstruct
+++ sunpinyin-2.0.4+git20160225.844ad0f/SConstruct
@@ -431,7 +431,7 @@ env.Substfile('sunpinyin-2.0.pc.in', SUB
     '@VERSION@': version,
     '@CFLAGS@': reduce(lambda a, b: a + ' ' + b,
                        map(lambda x: '-I$${includedir}' + x[3:],
-                           allinc())),
+                           sorted(allinc()))),
 })
 
 libname_default = '%ssunpinyin%s' % (env.subst('${SHLIBPREFIX}'),
